code/datums/brain_damage/phobia.dm

```diff
@@ -99,42 +196,65 @@
-			to_chat(owner, "<span class='warning'>You can't bring yourself to say the word \"<span class='phobia'>[word]</span>\"!</span>")
-			speech_args[SPEECH_MESSAGE] = ""
+			to_chat(owner, "<span class='warning'>Saying \"<span class='phobia'>[word]</span>\" puts you on edge!</span>")
+			if(fear_state <= (PHOBIA_STATE_CALM))
+				fearscore ++ 
 
-/datum/brain_trauma/mild/phobia/proc/freak_out(atom/reason, trigger_word)
-	next_scare = world.time + 120
-	if(owner.stat == DEAD)
+/datum/brain_trauma/mild/phobia/proc/freak_out(atom/reason, trigger_word, spooklevel = 0)//spooklevel is only used when calculating amount of scary items on a person.
+	if(owner.stat >= UNCONSCIOUS)
 		return
-	var/message = pick("spooks you to the bone", "shakes you up", "terrifies you", "sends you into a panic", "sends chills down your spine")
+	if(fear_state >= PHOBIA_STATE_EDGY)
+		stress_check = world.time + 3000
+		last_scare = world.time + 100
 	if(reason)
-		to_chat(owner, "<span class='userdanger'>Seeing [reason] [message]!</span>")
-	else if(trigger_word)
-		to_chat(owner, "<span class='userdanger'>Hearing \"[trigger_word]\" [message]!</span>")
+		if(isliving(reason))
+			var/mob/living/L = reason
+			if(spooklevel)
+				if(L.stat)
+					if(fear_state <= (PHOBIA_STATE_EDGY))
+						fearscore += spooklevel
+				else
+					fearscore += spooklevel * 2
+			else if(L.stat)
+				if(fear_state <= (PHOBIA_STATE_EDGY))
+					fearscore += 2
+			else
+				fearscore += 4//this is for simplemobs, but also dependent on species
+
+		else if(fear_state <= (PHOBIA_STATE_CALM)) //inanimate objects won't trigger you too much
+			fearscore ++
+		fearscore += stress
 	else
-		to_chat(owner, "<span class='userdanger'>Something [message]!</span>")
-	var/reaction = rand(1,4)
-	switch(reaction)
-		if(1)
-			to_chat(owner, "<span class='warning'>You are paralyzed with fear!</span>")
-			owner.Stun(70)
-			owner.Jitter(8)
-		if(2)
-			owner.emote("scream")
-			owner.Jitter(5)
-			owner.say("AAAAH!!", forced = "phobia")
+		fearscore ++ //I have no idea how this would happen. just increase fear by one, with no cap
+	switch(fear_state)//only happens once every five or so seconds, while scared
+		if(PHOBIA_STATE_EDGY)
+			owner.Jitter(1)
 			if(reason)
-				owner.pointed(reason)
-		if(3)
-			to_chat(owner, "<span class='warning'>You shut your eyes in terror!</span>")
-			owner.Jitter(5)
-			owner.blind_eyes(10)
-		if(4)
-			owner.dizziness += 10
-			owner.confused += 10
-			owner.Jitter(10)
-			owner.stuttering += 10
-
+				to_chat(owner, "<span class ='warning'>[reason] sets you on edge...</span>")
+		if(PHOBIA_STATE_UNEASY)
+			owner.Jitter(1)
+			if(reason)
+				to_chat(owner, "<span class ='warning'>[reason] makes you uneasy...</span>")
+		if(PHOBIA_STATE_FIGHTORFLIGHT)
+			owner.adjustStaminaLoss(-10 * (min(1, spooklevel)))
+			owner.SetUnconscious(0)
+			owner.SetStun(0)
+			owner.SetKnockdown(0)
+			owner.SetImmobilized(0)
+			owner.SetParalyzed(0)
```

Comment:
They get complete stun protection from being fearful? This seems like a great way to find a way to abuse for some temporary godlike immunity to sec.
Combined with the handcuff force break that is complete unstoppable this will make it so the second you see that message, you just have to beat the prisoner to death.

Reply:
The stamina regen makes sense but completely removing stuns (and becoming conscious again when unconscious especially), not so much

Reply:
ever woken up due to a nightmare? know how all other adrenaline chems also remove stuns?